### DISCUSSION TITLE
Is there a way to assist with transitionning UbuntuMATE to Meson without doing everything thru GitHub?

### DISCUSSION THREAD

**Discussion by ericmarceau-rogers** [mate-desktop/pluma#701] [General]
I would be willing to assist in trying to convert installation scripts to the new Meson system.  

I would be willing to work thru one of your "experts" passing along the converted files, piecemeal, starting from simpler components and gradually tackling more and more complex elements.

My experience with Makefiles is dated (20 years old) but my approach to tackling a process is demonstrated by my efforts and [**published script**](https://ubuntu-mate.community/t/build-customized-mate-themes-from-package-source/28104/2).

I would welcome this opportunity to assist the community.

I was thinking that Pluma might be a simple enough component, in comparison to others, for me to offer my assistance in this fashion.

Please advise whether, and how, you wish to proceed from here.  Thank you.

**Comment by lukefromdc** [comment]
I don't think that would be possible short of drop-in edited and new files posted to another site. Note that in my case I cannot access things like Facebook and Google, or anything that blocks content until ads or trackers are allowed.
 I myself do not have the time or energy to take on that sort of project and right now the team is shorthanded. All that hand editing would mean chasing down typos etc, and I have a limited understanding of meson myself
